fury/ui/containers.py

```diff
@@ -52,17 +62,42 @@ def _setup(self):
         """Setup this UI component.
 
         Create the background (Rectangle2D) of the panel.
-
+        Create the borders (Rectangle2D) of the panel.
         """
         self._elements = []
         self.element_offsets = []
         self.background = Rectangle2D()
+
+        if self.has_border:
+            self.borders = {'left': Rectangle2D(),
+                            'right': Rectangle2D(),
+                            'top': Rectangle2D(),
+                            'bottom': Rectangle2D()}
+
+            self.border_coords = {'left': (0., 0.),
+                                  'right': (1., 0.),
+                                  'top': (0., 1.),
+                                  'bottom': (0., 0.)}
+        else:
+            self.borders, self.border_coords = {}, {}
+
```

Comment:
No need of these member variables if the other blocks are moved within the `if block`.

Reply:
Done with [c43adff](https://github.com/fury-gl/fury/pull/441/commits/c43adff5b8f7f21d539bf58dc8e2f12d4d0cf640).